src/iocore/cache/Cache.cc

```diff
@@ -1810,7 +1810,7 @@ rebuild_host_table(Cache *cache)
 
 // if generic_host_rec.vols == nullptr, what do we do???
```

Comment:
nit: gen_host_rec`.stripes` instead of `.vol`?

Reply:
Nice catch! Done.